docs/releases/1.10.txt

```diff
@@ -323,6 +323,20 @@ added in Django 1.9::
 
 This prevents confusion about an assignment resulting in an implicit save.
 
+:mod:`django.contrib.gis`
+~~~~~~~~~~~~~~~~~~~~~~~~~
+* ``get_srid`` and ``set_srid`` methods of
```

Comment:
"The get_srid() and ... in favor of the ..." (similar comments for other notes). Also please add notes to docs/internals/deprecation.txt.